addons/resource.language.en_gb/resources/strings.po

```diff
@@ -8640,8 +8655,10 @@ msgctxt "#19056"
 msgid "New timer"
 msgstr ""
 
+#. Label for a context menu entry to open timer settings dialog
+#: xbmc/pvr/windows/GUIWindowPVRTimers.cpp
 msgctxt "#19057"
-msgid "Edit timer"
+msgid "Settings"
```

Comment:
I've seen that the dialog header is called "Timer settings", I still wouldn't associate "settings" with a way to edit the timer though. Once used to our terminology maybe, but as new user for sure not. What about a "edit settings" or "adjust timer"? That would at least make it clear that this context item triggers a editing mask.
Any preferences or opinions @jjd-uk?

And agreed about the system settings context item - IIRC that has been removed recently (or at least someone said he would remove it).